client/components/dialog/dialog-base.jsx

```diff
@@ -24,13 +29,16 @@ class DialogBase extends Component {
 			);
 
 		return (
-			<Modal isOpen={ this.props.isVisible }
+			<Modal
+				isOpen={ this.props.isVisible }
 				onRequestClose={ this._close }
```

Comment:
Potentially... I have seen other listeners in the codebase that listen for the escape key so maybe we _could_ see occasions where both are fired... I wonder if there would be any argument for letting the event through - I can't think of any myself.
I'll do some manual testing with preventDefault in there and feedback :)